packages/codecs/src/cacao.ts

```diff
@@ -47,8 +47,9 @@ export const CacaoSignature = strict(
   'CacaoSignature'
 )
 export type CacaoSignature = {
-  t: 'eip191' | 'eip1271' | 'solana:ed25519' | 'tezos:ed25519' | 'stacks:secp256k1'
+  t: 'eip191' | 'eip1271' | 'solana:ed25519' | 'tezos:ed25519' | 'stacks:secp256k1' | 'webauthn:p256'
   s: string
+  aad?: string
```

Comment:
To verify a webauthn signature we need to store Additional Authenticator Data.
I added prop `aad` to `CacaoSignature` as a string was that ok?